arm-oraclelinux-wls-cluster/src/main/arm/mainTemplate.json

```diff
@@ -527,6 +527,12 @@
                     "location": {
                         "value": "[parameters('location')]"
                     },
+                    "managedServerPrefix": {
+                        "value": "[parameters('managedServerPrefix')]"
+                    },
+                    "numberOfInstances": {
+                        "value": "[parameters('numberOfInstances')]"
+                    },
```

Comment:
Add managed parameters, used to update aad ip, import cer and enable tlsv1.2 in managed vms with vm extension.